core/src/com/unciv/logic/map/tile/Tile.kt

```diff
@@ -560,6 +560,15 @@ class Tile : IsPartOfGameInfoSerialization {
     fun getTilesInDistanceRange(range: IntRange): Sequence<Tile> = tileMap.getTilesInDistanceRange(position, range)
     fun getTilesAtDistance(distance: Int): Sequence<Tile> =tileMap.getTilesAtDistance(position, distance)
 
+    fun getAirUnitCapacity(): Int {
+        val airUnitCapacity = this.getCity()!!.civ.gameInfo.ruleset.modOptions.constants.airUnitCapacity
```

Comment:
Don't assume we have a city! In canPlaceNewUnit you can call this on a non-city tile!

Reply:
If the entire function is based on cities only, this should be a function in the *city* - but instead maybe it should contain logic for both city and non-city (carriers) and this remain here? In second thought that makes a lot of sense

Reply:
Perhaps I should just move this function to city ?
Because we already have the unique for carrier

![IMG_20240412_103543.jpg](https://github.com/yairm210/Unciv/assets/71725118/2ca533ec-e02b-442a-83df-66b174208540)

Reply:
That works